src/api/fetch.ts

```diff
@@ -1,5 +1,5 @@
 import { IModule, Module } from "../module.ts";
-import { ENDPOINT, apiFetch } from "./common.ts";
+import { apiFetch, ENDPOINT } from "./common.ts";
```

Comment:
Love how you noticed this and alphabetized it

Reply:
😄